packages/api/src/controllers/stream.ts

```diff
@@ -235,6 +236,39 @@ async function triggerManyIdleStreamsWebhook(ids: string[], queue: Queue) {
   );
 }
 
+async function resolvePullRegion(
+  stream: NewStreamPayload,
+  ingest: string
+): Promise<string> {
+  if (process.env.NODE_ENV === "test") {
+    return null;
+  }
+  const url = new URL(
+    pathJoin(ingest, `hls`, "not-used-playback", `index.m3u8`)
```

Comment:
don't we need to handle flv as well?

Reply:
No, this call is not for the "any redirect", this is just to discover which should be the ingest node.

So, kind-of hack the system and use catalyst-api redirect balancing (effectively using `MistUtilLoad`) to divide on the ingest node.